### PR TITLE
Wagtail upgrade

### PR DESCRIPTION
**Pull request by easherma**
Fixes https://github.com/cityofaustin/techstack/issues/2729
Also redirects straight to login cause that was annoying me

### CODE REVIEW

File: joplin/urls.py [hunk 1]
```diff
@@ -18,13 +18,19 @@ def home(request):
     return redirect('wagtailadmin_explore', page.id)
 
 
+def login(request):
+    return redirect(reverse('wagtailadmin_login'), permanent=True)
```
Comment (Niicck) [inline joplin/urls.py]: yay

File: joplin/urls.py [hunk 2]
```diff
@@ -18,13 +18,19 @@ def home(request):
...
+
+
 urlpatterns = [
     url(r'^django-admin/', include('smuggler.urls')),
     url(r'^django-admin/', admin.site.urls),
     path('admin/docs/', include('django.contrib.admindocs.urls')),
     path('admin/', home),
+    path('', login),
```
Comment (Niicck) [inline joplin/urls.py]: yay

File: joplin/templates/wagtailadmin/pages/listing/_table_headers_explore.html [hunk 3]
```diff
@@ -1,78 +1,63 @@
...
-            <a href="{% url 'wagtailadmin_explore' 3 %}?ordering={% if ordering == 'live' %}-{% endif %}live" class="icon icon-arrow-{% if ordering == '-live' %}up-after{% else %}down-after{% endif %} {% if ordering == 'live' or ordering == '-live' %}teal {% endif %}">
-                {% trans 'Status' %}
-            </a>
-        {% else %}
-            {% trans 'Status' %}
-        {% endif %}
+        {% trans 'Status' as status_label %}
+        {% page_table_header_label label=status_label sortable=sortable sort_field='live' parent_page_title=parent_page.title %}
     </th>
+    <th></th>
```
Comment (Niicck) [inline joplin/templates/wagtailadmin/pages/listing/_table_headers_explore.html]: ?

File: joplin/css/editor.scss [hunk 4]
```diff
@@ -1,10 +1,13 @@
...
+// maybe there is other niceness we are missing out on from the refactor
+// by just hiding these 
+.object .title-wrapper {
     background: inherit;
     text-transform: none;
 }
-.object > h2:before,
-.object > h2 label:before {
+.object > .title-wrapper:before,
+.object > .title-wrapper label:before {
```
Comment (Niicck) [inline joplin/css/editor.scss]: yay

File: joplin/templates/wagtailadmin/pages/listing/_table_headers_explore.html (hunk 3 above)
Comment (easherma) [reply joplin/templates/wagtailadmin/pages/listing/_table_headers_explore.html]: Oops. I should double check that this template overwrite is even necessary, I think I missed it from troubleshooting. Thanks for pointing it out!

Comment (easherma) [reply joplin/templates/wagtailadmin/pages/listing/_table_headers_explore.html]: I pulled it out of this PR

**Pull request merged**